test/Sema/accessibility_package_contextual_keyword.swift

```diff
@@ -0,0 +1,64 @@
+// RUN: %empty-directory(%t)
+// RUN: %{python} %utils/split_file.py -o %t %s
```

Comment:
General comment, this is the old version of the tool. For new files I'd recommend to use the following which is a bit cleaner:
```
// RUN: split-file %s %t
//--- File.swift
```

Reply:
sg